run_daemons.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Start all daemons
+
+nic=$1
+
+if [ "$1" == "-h" ]; then
+        echo "Usage: $0 <network interface>"
+        echo "   eg: $0 eth1"
+        exit
+fi
+
+if [ "$1" == "" ]; then
+        nic="eth1" #edit as required
```

Comment:
Note for future improvement: It is helpful to have script parameters like this in variables that are defined at the start of the file, which makes it obvious what parameters can be edited and puts them all together.